test/e2e/common/security_context.go

```diff
@@ -124,7 +124,7 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			framework.SkipIfNodeOSDistroIs("windows")
 			name := "explicit-nonroot-uid"
 			pod := makeNonRootPod(name, rootImage, pointer.Int64Ptr(1234))
-			pod = podClient.Create(pod)
+			_ = podClient.Create(pod)
```

Comment:
```suggestion 
			podClient.Create(pod)
```
Is the left side of the assignment needed here? (same for Line 147)

Reply:
@ahmad-diaa  right . i will delete them